### ISSUE TITLE
TD4a won't render

### ISSUE THREAD

**Issue by clay584** [cidrblock/td4a#15]
Just spun up your docker container but hitting render does nothing. Then after that, if I hit refresh, it no longer loads the demo content as it did on the first visit. I have to delete the cookies that were set by the app. It does work on your demo site though. I would love to use this, but I am not much help on debugging a webapp.

    docker pull cidrblock/td4a

    docker run -p 5000:5000 cidrblock/td4a

Using Chrome Version 68.0.3440.106 (Official Build) (64-bit).

Also tried Firefox 61.0.2 (64-bit) with the same behavior.

**Comment by clay584** [comment]
Turns out the dockerfile is pip installing 1.7.

**Comment by clay584** [comment]
pip installed 1.8 and the bug is still there. Can you publish 1.9 with HEAD on master branch so the bug will be resolved?

**Comment by danielolson13** [comment]
Rendering issue with the version on pypi is because controllers/render.py has an unknown keyword arg "type"

```
def jinja_unresolved(template, typ):
...
if jinja_unresolved(template=after_jinja, type="p1"):
```
Should be...
```
def jinja_unresolved(template, typ):
...
if jinja_unresolved(template=after_jinja, typ="p1"):
```

**Comment by asmanea** [comment]
Had the same issue with virtualenv, fixed the above line mentioned by danielolson13, now template renders.

**Comment by cidrblock** [comment]
New version released, everything up to 2.0.3 now.
